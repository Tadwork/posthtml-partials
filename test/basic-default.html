<mixin name="say" class from="me">
	<p class="{{ class }}">Hello from {{ from }}!</p>
</mixin>

<div>
	<mixin name="say" class="hello"></mixin>
</div>
