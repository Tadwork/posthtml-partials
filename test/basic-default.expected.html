

<div>
	
	<p class="hello">Hello from me!</p>

</div>
